src/common/lib/fs.js

```diff
@@ -0,0 +1,5 @@
+import {promisifyAll} from 'bluebird';
+
+const fs = promisifyAll(require('fs'));
+
+export default fs;
```

Comment:
You can't access `fs` from RN due to iOS runtime (and Android as well). There can be `fs` plugin that proxies requests to iOS file-system, but that's out of scope for that file.